README.md

```diff
@@ -1,16 +1,55 @@
 foxsi-science
 =============
 
+INSTALL:
+
+    You need to have IDL and SSW previously installed
+
+    Include this five lines in your personal IDL_STARTUP file
+
+    COMMON FOXSI,FOXSIPKG,FOXSIDB
+    FOXSIPKG='path/to/your/foxsi-science'
+    FOXSIDB='path/to/your/foxsi-data/directory/foxsidb'
+    add_path,FOXSIPKG,/expand
+    add_path,FOXSIDB
+    restore,FOXSIDB+'/data_2014/flight2014-parameters.sav'
+
```

Comment:
Using the alternate method we discussed, lines 10-15 would be replaced with:

setenv, 'FOXSIPKG=/path/to/your/foxsi-science'
setenv, 'FOXSIDB=/path/to/your/foxsi-data/directory/foxsidb'
add_path, '$FOXSIPKG', /expand
add_path, '$FOXSIDB'
restore, '$FOXSIDB/data_2014/flight2014-parameters.sav'